jax/lax.py

```diff
@@ -843,14 +843,16 @@ def _brcast(x, *others):
 def _brcast_to(x, shape):
```

Comment:
Can you add a one sentence doc comment describing what this function does?

I'm trying to figure out why the first case is correct — if the intent is that this function returns a tensor with shape `shape`, surely it won't here? I guess this is doing something for zero-element tensors but I am unsure exactly what and not completely sure what this function is supposed to do.

Reply:
Done! I also made it better by calling a function you wrote :)